Iteam1337/sonos-wejay
# feat: add ability to receive slack-token

This adds the ability to retrieve your `access_token` via Slack OAuth.

**How to test:**

You'll need to add the following to your .env-file (they are both available at Basic Settings in the Slack App-page)

```
SLACK_CLIENT_ID
SLACK_CLIENT_SECRET
```

Go to `http://SERVER_URL/slack/auth`, you should be redirected to Slack's OAuth. After you've authorized Wejay, you should be redirected to `https://SERVER_URL/slack/token` where your token is available as a string.

If you pass your `access_token` in the headers of the HTTP-request, we'll be able to get the user-data from `Slack.getUser` and pass along the `userId`.
